ppafm/HighLevel.py

```diff
@@ -146,7 +146,7 @@ def computeLJ( geomFile, speciesFile, save_format=None, computeVpot=False, Fmax=
     FFparams            = PPU.loadSpecies( speciesFile )
     elem_dict           = PPU.getFFdict(FFparams); # print elem_dict
     # --- load atomic geometry
-    atoms,nDim,lvec     = BU.loadGeometry( geomFile, params=PPU.params )
+    atoms,nDim,lvec     = BU.loadGeometry( geomFile, params=PPU.params ) # atoms can be iether [e,x,y,z] or [e,x,y,z,q] for each atoms #
```

Comment:
What is the purpose of the `#` symbol in the terminal position?
```suggestion
    atoms,nDim,lvec     = BU.loadGeometry( geomFile, params=PPU.params ) # atoms can be iether [e,x,y,z] or [e,x,y,z,q] for each atoms.
```

Reply:
It is just my way, as I use comments ...